types/validator_set_test.go

```diff
@@ -664,88 +664,189 @@ type testVal struct {
 	power int64
 }
 
-func TestValSetUpdatesBasicTestsExecute(t *testing.T) {
-	valSetUpdatesBasicTests := []struct {
-		startVals    []testVal
-		updateVals   []testVal
-		expectedVals []testVal
-		expError     bool
-	}{
-		// Operations that should result in error
-		0: { // updates leading to overflows
```

Comment:
I'd leave the indexes (0, 1, ...). If a test case fails, it's easy to locate it by searching for the index.